Lib/test/test_logging.py

```diff
@@ -3928,6 +3928,50 @@ def test_config_queue_handler(self):
             msg = str(ctx.exception)
             self.assertEqual(msg, "Unable to configure handler 'ah'")
 
+    @threading_helper.requires_working_threading()
+    @support.requires_subprocess()
+    @patch("multiprocessing.Manager")
+    def test_config_queue_handler_does_not_create_multiprocessing_manager(self, manager):
+        # gh-120868
+
+        from multiprocessing import Queue as MQ
+
+        q1 = {"()": "queue.Queue", "maxsize": -1}
+        q2 = MQ()
+        q3 = queue.Queue()
+
+        for qspec in (q1, q2, q3):
+            self.apply_config(
+                {
+                    "version": 1,
+                    "handlers": {
+                        "queue_listener": {
+                            "class": "logging.handlers.QueueHandler",
+                            "queue": qspec,
+                        },
+                    },
+                }
+            )
+            manager.assert_not_called()
+
+    @patch("multiprocessing.Manager")
+    def test_config_queue_handler_invalid_config_does_not_create_multiprocessing_manager(self, manager):
```

Comment:
I'd also add a test where `MM().Queue()` fails and check that you have your TypeError.

Reply:
This would only happen if `.configure_handler` is called directly, since otherwise `.configure` wraps all errors occurring inside `.configure_handler` in a `ValueError`. So far this isn't being tested directly in `test_logging`, which I assumed was intentional. If that's not the case, I'm happy to add a test for this :)

Reply:
Oh I see! Then I think it's fine like that.